template-manager-blueprint/src/main/resources/definitions/cloud-storage-location-specification.json

```diff
@@ -3,140 +3,110 @@
     {
       "propertyName": "logfeeder.cloud.storage.base.path",
       "propertyFile": "logfeeder-properties",
-      "description": "This is the directory where cluster daemon logs should be stored by Log Feeder",
+      "description": "This is the directory where cluster daemon logs will be stored by Log Feeder",
       "defaultPath": "{{{ defaultPath }}}/clusterlogs",
       "propertyDisplayName": "Cluster Daemon Logs",
-      "relatedService": "LOGSEARCH_LOGFEEDER",
+      "relatedServices": ["LOGSEARCH_LOGFEEDER"],
       "requiredForAttachedCluster": false,
       "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
     },
     {
       "propertyName": "zeppelin.notebook.dir",
       "propertyFile": "zeppelin-site",
-      "description": "The root directory where Zeppelin notebook directories are saved",
-      "defaultPath": "{{{ defaultPath }}}/spark/zeppelin/notebook",
+      "description": "The directory where Zeppelin notebooks are saved",
+      "defaultPath": "{{{ defaultPath }}}/zeppelin/notebook",
       "propertyDisplayName": "Zeppelin Notebooks Root Directory",
-      "relatedService": "ZEPPELIN_MASTER",
+      "relatedServices": ["ZEPPELIN_MASTER"],
       "requiredForAttachedCluster": false,
       "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
     },
     {
       "propertyName": "tez.history.logging.proto-base-dir",
       "propertyFile": "tez-site",
-      "description": "The directory into which Tez history data will be written when proto history logging service is used",
-      "defaultPath": "{{{ defaultPath }}}/apps/hive/ewarehouse/tablespace/external/hive/sys.db/query_data",
+      "description": "The directory into which Tez/Hive job history data will be written",
+      "defaultPath": "{{{ defaultPath }}}/warehouse/tablespace/external/hive/sys.db/query_data",
```

Comment:
Please note that due to BUG-115324 `{{{ defaultPath }}}` now never contains the cluster name. If that is required anywhere for the demo, `/{{{ clusterName }}}` or `{{{#if-false attachedCluster}}}/{{{ clusterName }}}{{{/if-false}}}` should be added as appropriate. Sorry for the obfuscation.

Reply:
Looks fine. Nice job!